async function main() {}

main().catch(e => console.error(e));
